src/NumberToWords.php

```diff
@@ -70,7 +72,8 @@ class NumberToWords
         'sv' => SwedishNumberTransformer::class,
         'tk' => TurkmenNumberTransformer::class,
         'tr' => TurkishNumberTransformer::class,
-        'ua' => UkrainianNumberTransformer::class
+        'ua' => UkrainianNumberTransformer::class,
+        'ka' => GeorgianNumberTransformer::class
```

Comment:
Keys in the array are sorted alphabetically. Could you please move the `'ka' => GeorgianNumberTransformer::class` above?